### ISSUE TITLE
openseedbox.assets.prefix

### ISSUE THREAD

**Issue by psycholyzern** [erindru/openseedbox#22]
default value is ```openseedbox.assets.prefix=https://s3.amazonaws.com/cdn.openseedbox.com```
Can I know what it does actually?

**Comment by erindru** [comment]
Thats actually a fail on my part, keeping it in the config. You notice if you "view-source" in the UI, all CSS/images etc will be prefixed with that.

It should actually be set to your context root, so "/" in the default setup. Handily, all the Openseedbox CSS/Javascript assets are already uploaded there, because I use it for my personal instance.

The reason it exists is that on very early versions of Openseedbox, I was trying to get it to run on Heroku and its recommended to serve static assets from a CDN or something like Amazon S3 to take load off the Heroku instance.

**Comment by psycholyzern** [comment]
I tried. But the webpage broken.
![](http://i.imgur.com/XuMhiHD.png)

 Seems like js/css/images did not showing right. I tried to access an image and got only codes and symbols displayed in my browser. But when I change to your amazon instances, the webpage back to normal.
![](http://i.imgur.com/JhHpgLk.png)

I alias ```/src/openseedbox``` to ```/res```
```
location /res {
  alias /src/openseedbox
```

Then I set 
```
openseedbox.assets.prefix=http://example.com/res/public
```

I also noticed errors in play! log ```/src/openseedbox/logs/system.out```
```
CompilerOracle: exclude jregex/Pretokenizer.next
Listening for transport dt_socket at address: 8000
19:09:12,628 INFO  ~ Starting /src/openseedbox
19:09:12,638 INFO  ~ Module openseedbox is available (/src/openseedbox/../openseedbox-common)
19:09:12,639 INFO  ~ Module siena is available (/src/openseedbox/modules/siena-2.0.7)
19:09:14,979 WARN  ~ You're running Play! in DEV mode
19:09:15,440 INFO  ~ Listening for HTTP on port 9000 (Waiting a first request to start) ...
19:33:31,711 DEBUG ~ Play-Siena: enhancing EnhancedModel com.openseedbox.models.Torrent
19:33:32,090 DEBUG ~ Play-Siena: enhancing EnhancedModel com.openseedbox.models.ModelBase
19:33:33,106 DEBUG ~ Play-Siena: enhancing EnhancedModel com.openseedbox.models.Node
19:33:33,962 DEBUG ~ Play-Siena: enhancing EnhancedModel com.openseedbox.models.EventBase
19:33:34,066 DEBUG ~ Play-Siena: enhancing EnhancedModel com.openseedbox.models.UserMessage
19:33:34,188 DEBUG ~ Play-Siena: enhancing EnhancedModel com.openseedbox.models.UserTorrent
19:33:34,389 DEBUG ~ Play-Siena: enhancing EnhancedModel com.openseedbox.models.NodeGroup
19:33:34,548 DEBUG ~ Play-Siena: enhancing EnhancedModel com.openseedbox.models.TorrentEvent
19:33:36,016 DEBUG ~ Play-Siena: enhancing EnhancedModel com.openseedbox.models.InvoiceLine
19:33:36,119 DEBUG ~ Play-Siena: enhancing EnhancedModel com.openseedbox.models.JobEvent
19:33:37,956 DEBUG ~ Play-Siena: enhancing EnhancedModel com.openseedbox.models.Invoice
19:33:38,757 DEBUG ~ Play-Siena: enhancing EnhancedModel com.openseedbox.models.EmailError
19:33:38,828 DEBUG ~ Play-Siena: enhancing EnhancedModel com.openseedbox.models.User
19:33:38,876 DEBUG ~ Play-Siena: enhancing EnhancedModel com.openseedbox.models.Plan
19:33:55,088 INFO  ~ Connected to jdbc:mysql://localhost/openseedbox
19:33:55,381 INFO  ~ Siena DB Type: sql:mysql
19:33:55,459 DEBUG ~ Siena DDL Type: mysql
19:33:56,731 DEBUG ~ Siena DDL dev mode: update
19:33:58,910 DEBUG ~ Siena DDL Generator SQL:
19:33:58,964 INFO  ~ Executed 0 SQL command(s) with 0 error(s)
19:33:59,049 INFO  ~ Application 'openseedbox' is now started !
19:33:59,510 ERROR ~

@6m414po68
POST_ip_port.php.index action not found

Action not found
Action POST_ip_port.php.index could not be found. Error raised is Controller controllers.POST_ip_port.php not found

play.exceptions.ActionNotFoundException: Action POST_ip_port.php.index not found
        at play.mvc.ActionInvoker.getActionMethod(ActionInvoker.java:599)
        at play.mvc.ActionInvoker.resolve(ActionInvoker.java:85)
        at Invocation.HTTP Request(Play!)
Caused by: java.lang.Exception: Controller controllers.POST_ip_port.php not found
        ... 3 more

```

**Comment by erindru** [comment]
Sorry, it should be /public, not /

Just take out the `%prod.openseedbox.assets.prefix` line from the config file, and remove your nginx alias
